apps/website/src/components/shared/form/ImageUploadAttachment.tsx

```diff
@@ -13,10 +13,12 @@ export function ImageUploadAttachment({
   removeFileReference,
   fileReference,
   children = "",
+  onClick,
 }: {
   removeFileReference: (id: string) => void;
   fileReference: FileReference;
   children?: ReactNode | ReactNode[];
+  onClick?: () => void;
 }) {
   const src =
     fileReference.status === "saved"
```

Comment:
@pjeweb 
For your suggestion to work we will need to also update the image source like this. Otherwise we will get the `Not allowed to navigate top frame to data URL` error because we're using a data url in the anchor link.
```suggestion
    fileReference.status === "saved" || fileReference.status === "upload.done"
```

Reply:
That seems fine, yeah. Sorry for not responding earlier. I really only care about the review, but yeah ideally it should also work immediately after the upload. I think I did not use the uploaded url so its not having to fetch the image again (that was just uploaded), but it does make sense!

Reply:
@pjeweb 
I updated the code to support command clicking.